mfhdf/hrepack/hrepacktst.c

```diff
@@ -2089,15 +1440,11 @@ add_pal(const char *fname)
  *
  * Return: SUCCEED, FAIL
  *
- * Programmer: Pedro Vicente, pvn@ncsa.uiuc.edu
- *
- * Date: September 15, 2003
- *
  *-------------------------------------------------------------------------
  */
+#ifdef H4_HAVE_LIBSZ
```

Comment:
No reason to have this function visible at all when there is no szip. The call to it is hidden by ifdef guards, so compiling it is a waste of time and generates warnings.